package.json

```diff
@@ -14,9 +14,9 @@
     "build": "yarn build-sass && yarn build-js && yarn copy-assets",
     "build-sass": "node-sass app/assets/sass/application.scss public/css/main.css --output-style compressed",
     "copy-assets": "cp -R ./node_modules/govuk-frontend/assets ./public/govuk-frontend",
-    "build-js": "browserify ./app/assets/javascript/main.js -o ./public/js/main.js",
+    "build-js": "browserify ./app/assets/javascript/main.ts -p [ tsify --noImplicitAny ] > ./public/js/main.js",
     "test": "yarn test:unit && yarn test:browser",
-    "test:unit": "cross-env NODE_PATH=. NODE_ENV=test mocha test/unit --recursive",
+    "test:unit": "cross-env NODE_PATH=. NODE_ENV=test mocha  -r ts-node/register test/unit/**/*.test.ts --recursive",
```

Comment:
Doing this means that we are not running the JS unit tests, might need to have a separate script for running the typescript ones or something?

Reply:
👍 totally agree on the aim, but it might take a bit of time.